com.slack/device.js

```diff
@@ -38,8 +38,8 @@ module.exports = class InstagramClass extends Tp.BaseDevice {
 
     constructor(engine, state) {
         super(engine, state);
-        this.uniqueId = 'com.slack-' + this.state.user_id;
-        this.name = "Slack %s".format(this.state.user_id);
+        this.uniqueId = 'com.slack-' + this.state.user;
```

Comment:
The uniqueId should probably still use the user_id, as the username might change but Thingpedia unique ID are supposed to be immutable.